src/main/kotlin/no/skatteetaten/aurora/boober/feature/BigIpLegacyFeature.kt

```diff
@@ -0,0 +1,90 @@
+package no.skatteetaten.aurora.boober.feature
+
+import com.fkorotkov.kubernetes.newObjectMeta
+import no.skatteetaten.aurora.boober.model.AuroraConfigFieldHandler
+import no.skatteetaten.aurora.boober.model.AuroraContextCommand
+import no.skatteetaten.aurora.boober.model.AuroraDeploymentSpec
+import no.skatteetaten.aurora.boober.model.AuroraResource
+import no.skatteetaten.aurora.boober.model.openshift.BigIp
+import no.skatteetaten.aurora.boober.model.openshift.BigIpKonfigurasjonstjenesten
+import no.skatteetaten.aurora.boober.model.openshift.BigIpSpec
+import no.skatteetaten.aurora.boober.utils.addIfNotNull
+import org.apache.commons.codec.digest.DigestUtils
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+
+@Service
+class BigIpLegacyFeature(
+    @Value("\${boober.route.suffix}") val routeSuffix: String,
+    val bigIpFeature: BigIpFeature
+) : Feature {
+
+    override fun enable(header: AuroraDeploymentSpec): Boolean {
+        return bigIpFeature.enable(header)
+    }
+
+    override fun handlers(header: AuroraDeploymentSpec, cmd: AuroraContextCommand): Set<AuroraConfigFieldHandler> {
+        return bigIpFeature.handlers(header, cmd)
```

Comment:
Sånn egentlig får man dobbelt opp med handlers. Det blir kalt fra både bigiplegacyfeature og bigipfeature. Går det fint? Bør vi gjøre det? 
En av grunnene til at jeg er litt betent på bigipfeature og bigiplegacy feature er at det kjøres dobbelt opp av enable, handlers og validate, selv om det kanskje har kjørt før. I det store og det hele så koster det nok ikke så mye, men føles bare ikke helt 100. 
Men samtidig så ser jeg ikke en fullverdig god løsning på det heller, så kan være vi bare må ha det sånn siden det blir for komplisert å kombinere eller splitte de ut til separate.

Reply:
Den returnerte et set så det ville ikke blitt duplikater. Jeg måtte ha dem begge steder for å ikke måtte forholde meg til rekkefølge problemer, men har nå slått dem sammen.